srcpkgs/iverilog/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'iverilog'
 pkgname=iverilog
 version=10.2
-revision=1
+revision=2
 wrksrc="${pkgname}-${version/./_}"
 build_style=gnu-configure
 hostmakedepends="automake flex gperf"
-makedepends="readline-devel"
+makedepends="readline-devel zlib-devel"
 short_desc="Verilog simulation and synthesis tool"
 maintainer="Orphaned <orphan@voidlinux.eu>"
 license="GPL-2, LGPL-2.1"
```

Comment:
While you're at it, please fix these licenses to use SPDX identifier.

Reply:
done